website/docs/reference/dbt-jinja-functions/model.md

```diff
@@ -15,9 +15,42 @@ For example:
 {% endif %}
 ```
 
-To view the contents of `model` for a given model, use [log()](/reference/dbt-jinja-functions/log)
+To view the contents of `model` for a given model:
+
+<Tabs>
+
+<TabItem value="cli" label="CLI">
+
+If you're using the CLI, use [log()](/reference/dbt-jinja-functions/log)
 to print the full contents:
 
 ```jinja
 {{ log(model, info=True) }}
 ```
+  
+ </TabItem>
+ 
+ <TabItem value="ide" label="dbt Cloud IDE">
+   
+ If you're using the dbt Cloud IDE, compile the following to print the full contents:
+ 
+```jinja
+{{ model \\| tojson(indent = 4) }}
```

Comment:
@JKarlavige Any idea why this pipe is making netlify fail? I tried escaping it, but it still doesn't work :(

Reply:
Thanks  @JKarlavige Asana task opened: https://app.asana.com/0/1200099998847559/1204463347472924/f

Reply:
hey @patkearns10 - should there be a "," in the below code?

```jinja
{{ model | tojson(indent = 4) }}
```

Reply:
Nope! you are taking the output of the model variable and piping it into a function tojson():
<img width="569" alt="Screen Shot 2023-04-17 at 11 46 14 AM" src="https://user-images.githubusercontent.com/30663534/234434180-49b71317-95da-4dc3-9c97-d50c7ec7d500.png">